src/DynamoCoreWpf/Extensions/ViewLoadedParams.cs

```diff
@@ -57,6 +59,14 @@ public IPackageInstaller PackageInstaller
             get { return dynamoViewModel.PackageManagerClientViewModel; }
         }
 
+        /// <summary>
+        /// A reference to package loader to query for certain packages info
+        /// </summary>
+        public IPackageLoader PackageLoader
```

Comment:
It doesn't really make sense to me that this is only added to View extensions -

Reply:
You mean?

Reply:
You can add it to model extensions as well by adding it to `ReadyParams`